used jest and added .test file too

Prasiddhidahal opened the pull request:
![Screenshot 2023-11-05 160348](https://github.com/org-45/yantra-package/assets/87261517/e2f0e91f-686c-4b75-802e-d02a6483aea1)
I used jest tool and carried out promise test. I added .test.js too for testing and works fine. Screenshot attached. 

Closes https://github.com/org-45/yantra-package/issues/5

Prasiddhidahal committed:
used jest and added .test file too

manjilkoirala reviewed (commented):
code is good and working fine.
Just open a issue and link it to the PR. Also, use the format of description provided before.

Prasiddhidahal commented:
Description
To add contribution to yantra-package and worked with jest tool which is used for testing.

Changes Made
I have added 2 files called fetchData.js and fetchData.test.js where in .test file i have used jest function and tested the code and it works fine. 

Checklist 
 I have tested these changes locally.
 I have added/updated documentation as needed.
 I have reviewed my code for potential issues.
 I have followed the project's coding guidelines.
 I have updated relevant comments and commit messages.

Screenshot
![Screenshot 2023-11-05 160348](https://github.com/org-45/yantra-package/assets/87261517/8e297272-0353-4154-94a0-ec6b0e9cc527)
 

 By submitting this PR, I confirm that
 I have read and understood the project's contributing guidelines.
 This PR is ready for review and merging.
 I understand that automated tests should pass before merging.

manjilkoirala commented:
The description is good. Now just link the issue in the description then you are good to go.
